vm/uri.go

```diff
@@ -21,6 +22,14 @@ var builtinURIClassMethods = []*BuiltinMethodObject{
 		// ```
 		Name: "parse",
 		Fn: func(receiver Object, sourceLine int, t *Thread, args []Object, blockFrame *normalCallFrame) Object {
+			if len(args) != 1 {
+				return t.vm.InitErrorObject(errors.ArgumentError, sourceLine, errors.WrongNumberOfArgument, 1, len(args))
+			}
+			typeErr := t.vm.checkArgTypes(args, sourceLine, classes.StringClass)
```

Comment:
nitpick: can you add newlines before/after this line? we usually add newlines around the if statements for readability 😄

Reply:
Updated.